brave/brave-core
# Use flag images from leo for vpn panel

Resolves https://github.com/brave/brave-browser/issues/29727

![image](https://github.com/brave/brave-core/assets/6786187/cecc046e-2704-4bc1-a42c-577b4eb51ad0)


No functional change.



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

1. Set as purchased state
2. Launch vpn panel and check flags are rendered properly
